enrolments/tests/test_serializers.py

```diff
@@ -237,24 +242,31 @@ def setUp(self):
             "preferred_class": self.class2.id,
             "enrolled_class": self.class1.id,
             "status": Enrolment.CLASS_ALLOCATED,
+            "students": [self.family.parent.id],
         }
 
     def test_enrolment_serializer(self):
         self.assertEqual(
             {
                 "id": self.enrolment.id,
-                "session": {
-                    "id": self.session.id,
-                    "name": self.session.name,
-                },
-                "preferred_class": {
-                    "id": self.class1.id,
-                    "name": self.class1.name,
-                },
-                "enrolled_class": {
-                    "id": self.class2.id,
-                    "name": self.class2.name,
-                },
+                "session": SessionListSerializer(self.session).data,
+                "preferred_class": ClassListSerializer(self.class1).data,
+                "enrolled_class": ClassListSerializer(self.class2).data,
```

Comment:
this trick is pog. I feel like I've tried it before and couldn't make it work, but now I know it's legit